lib/src/util/css_value_util.dart

```diff
@@ -51,44 +51,44 @@ class CssValue implements Comparable<CssValue> {
   ///     20
   ///     '1.25em'
   ///     '-15%'
-  factory CssValue.parse(dynamic source, {CssValue onError(value, error)}) {
+  factory CssValue.parse(dynamic source, {CssValue onError(value, error)}) { // ignore: use_function_type_syntax_for_parameters
```

Comment:
#nit Any reason this was ignored instead of updated?
```suggestion
  factory CssValue.parse(dynamic source, {CssValue Function(dynamic value, dynamic error) onError}) {
```